README.md

```diff
@@ -1,2 +1,78 @@
 # serenity-appium-app-browserstack
-This repository demonstrates how to run Appium tests in Xunit on BrowserStack App Automate.
+
+[Serenity](https://serenity-bdd.info/) Integration with BrowserStack SDK.
+
+![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
+
+## Based on
+
+These code samples are currently based on:
+
+- **Appium-Java-Client:** `8.1.1`
+- **Protocol:** `W3C`
+
+## Setup
+
+### Requirements
+
+1. Java 8+
+
+    - If Java is not installed, follow these instructions:
+        - For Windows, download latest java version from [here](https://java.com/en/download/) and run the installer executable
+        - For Mac and Linux, run `java -version` to see what java version is pre-installed. If you want a different version download from [here](https://java.com/en/download/)
+
+2. Maven
+    - If Maven is not downloaded, download it from [here](https://maven.apache.org/download.cgi)
+    - For installation, follow the instructions [here](https://maven.apache.org/install.html)
+
+### Install the dependencies
+
+To install the dependencies, run :
+
+- For Android
+
+    ```sh
+    cd android
+    mvn clean
+    ```
+
+- For iOS
+
+    ```sh
+    cd ios
+    mvn clean
+    ```
+
+## Getting Started
+
+Getting Started with Appium tests on BrowserStack couldn't be easier!
```

Comment:
Don't we need the framework name here like we added in other repos? 
https://github.com/browserstack/cucumber-java-appium-app-browserstack/blob/ee69fd4d053d3eaf3c4d93cac77aeebfb849f382/README.md?plain=1#L40